lldb/test/API/lit.site.cfg.py.in

```diff
@@ -10,6 +10,7 @@ config.llvm_build_mode = lit_config.substitute("@LLVM_BUILD_MODE@")
 config.lit_tools_dir = "@LLVM_LIT_TOOLS_DIR@"
 config.lldb_obj_root = "@LLDB_BINARY_DIR@"
 config.lldb_src_root = "@LLDB_SOURCE_DIR@"
+config.lldb_built_include_dir = lit_config.substitute("@LLDB_BINARY_DIR@/include")
```

Comment:
If this is always `@LLDB_BINARY_DIR@/include`, why not use 

```
config.lldb_built_include_dir = os.path.join(config.lldb_obj_root. 'include')
```

or even better: don't create a new variable at all and have dotest derive it from `config.lldb_obj_root`?

Reply:
@JDevlieghere  I made the change — I'm not sure it does what you were hoping for though, since we still need to thread through `lldb_obj_root`.